docker/opensource.md

```diff
@@ -87,5 +87,102 @@ $ docker run -it --rm --workdir /opt -v$(pwd):/opt ubuntu:22.10 bash
 # phpunit
 ```
 
+## Cosmo-Tech
+{id: docker-python-Cosmo-Tech}
+
+
+* [Cosmo-Tech](https://github.com/Cosmo-Tech/CosmoTech-Acceleration-Library)
+
+```
+$ git clone https://github.com/Cosmo-Tech/CosmoTech-Acceleration-Library.git
+$ cd CosmoTech-Acceleration-Library
+$ docker run -t -i --name cosmotech-acceleration-library-dev -v /opt/CosmoTech-Acceleration-Library -v /C:/Users/YourUserName/CosmoTech-Acceleration-Library:/opt python:3.11 bash
+
+# cd opt
```

Comment:
if -w /opt is provided on the command line of docker run then there is no need for this.

Reply:
again it didn't worked for me